app/code/Codifi/CustomerRequest/Plugin/Cart/Item/RendererPlugin.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\CustomerRequest\Plugin\Cart\Item;
+
+/**
+ * Class RendererPlugin
+ * @package Codifi\CustomerRequest\Plugin\Cart\Item
+ */
+class RendererPlugin
+{
+    /**
+     * Check Product has URL
+     *
+     * @return bool
+     */
+    public function afterHasProductUrl(): bool
```

Comment:
Reed https://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html and correct your plugin